release/release.mk

```diff
@@ -22,37 +22,37 @@ ALL_ARCH = amd64 arm arm64 ppc64le s390x
 .PHONY: ko-release
 ko-release:
 # amd64
-	LDFLAGS="$(LDFLAGS)" GIT_HASH=$(GIT_HASH) GIT_VERSION=$(GIT_VERSION) \
-	ko publish --base-import-paths --bare \
-		--platform=linux/amd64 --tags $(GIT_VERSION)-amd64 --tags $(GIT_HASH)-amd64 \
-		github.com/sigstore/fulcio
+	ARCH="amd64" LDFLAGS="$(LDFLAGS)" GIT_HASH=$(GIT_HASH) GIT_VERSION=$(GIT_VERSION) \
+	ko resolve --base-import-paths \
+		--platform=linux/$(ARCH) --tags $(GIT_VERSION)-$(ARCH) --tags $(GIT_HASH)-$(ARCH) \
```

Comment:
the $(ARCH) does not get filled with the data, maybe there is another way to do that, but maybe we keep as the original for now, i was planning to do a refactor on this and use a for loop

Reply:
besides that i was able to generate the yaml :)

Reply:
thanks for finding that issue. i just dont understand shell sometimes.